src/plugins/experimental-discord/mirror.js

```diff
@@ -23,21 +23,21 @@ export class Mirror {
     this.guild = guild;
   }
 
-  async update(reporter: TaskReporter) {
+  async update(manager: TaskManager) {
```

Comment:
nit: `reporter` was a fairly self-descriptive variable name (it reports things), `manager` less so because it could be managing anything.

maybe we should call it `reporterManager`? or maybe the `TaskManager` class should itself be called `ScopedTaskReporter`?